go/cmd/api-gateway/requests.go

```diff
@@ -36,18 +40,35 @@ func requestHandler() http.HandlerFunc {
 			return
 		}
 
+		userPb := &pb.User{
+			Id:       reqApproval.User.Id,
+			UserName: reqApproval.User.UserName,
+		}
+
```

Comment:
Why'd you separate the user here? Doesn't really matter but makes it less logical to me